### PR TITLE
Add a google.datalab.ml.Metrics class, which computes accuracy, RMSE, MAE, Percentile from raw evaluation data.

### PR DESCRIPTION
**Pull request by qimingj**
…

### CODE REVIEW

File: google/datalab/ml/_metrics.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+        correct_count = len(df[(df['target'] == df['predicted']) & (df['target'] == label)])
+        total_count = len(df[(df['target'] == label)])
+        accuracy_results.append({
+            'target': label,
+            'accuracy': float(correct_count) / total_count,
+            'count': total_count
+        })
+
+      total_correct_count = len(df[(df['target'] == df['predicted'])])
+      total_accuracy = float(total_correct_count) / len(df)
```
Comment (chmeyers) [inline google/datalab/ml/_metrics.py]: Just double checking, is len(df) ever zero?

File: google/datalab/ml/_metrics.py [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+      number.
+    """
+
+    if self._input_csv_files:
+      df = self._get_data_from_csv_files()
+      if 'target' not in df or 'predicted' not in df:
+        raise ValueError('Cannot find "target" or "predicted" column')
+
+      df = df[['target', 'predicted']].apply(pd.to_numeric)
+      mse = mean_squared_error(df['target'], df['predicted'])
```
Comment (chmeyers) [inline google/datalab/ml/_metrics.py]: Is mse ever negative or a NaN?

File: google/datalab/ml/_metrics.py (hunk 1 above)
Comment (qimingj) [reply google/datalab/ml/_metrics.py]: Done. Also did similar check in a few other places, and tested 0 instance in csv and 0 instance in bq.

File: google/datalab/ml/_metrics.py (hunk 2 above)
Comment (qimingj) [reply google/datalab/ml/_metrics.py]: Tested it. If NaN, sklearn raises understandable error.

**Comment by qimingj** [comment]
PTAL.

**Pull request merged**